pkg/pronom/internal/mappings/droid.go

```diff
@@ -12,30 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package mappings contains struct mappings to unmarshal three
+// Package mappings contains struct mappings to unmarshall three
```

Comment:
While totally inconsequential, I just wanted to share my observation that the previous spelling was already the correct one (see https://en.wiktionary.org/wiki/unmarshal)?